UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -855,6 +855,18 @@
   }
 !endif
 
+  #
+  # Rng Protocol producer
+  #
+  SecurityPkg/RandomNumberGenerator/RngDxe/RngDxe.inf

```

Comment:
should these two be guarded by `!if $(NETWORK_DRIVER_ENABLE) == TRUE` as above / as in the FDF?

Reply:
Well, they would just be built. But I guess for having it standardized in both files it would be the best. I'll make the changes and update the PR.